diff --git a/Makefile.am b/Makefile.am
old mode 100644
new mode 100755
index 3d9e5c1..c7a512a
--- a/Makefile.am
+++ b/Makefile.am
@@ -69,7 +69,7 @@ dbussessionservicedir=@dbussessionservicedir@
 dbussystemservicedir=@dbussystemservicedir@
 pamlibdir=@pamlibdir@
 pamconfdir=@pamconfdir@
-pkgconfigdatadir=$(datadir)/pkgconfig
+pkgconfigdatadir=$(libdir)/pkgconfig
 pkgconfiglibdir=$(libdir)/pkgconfig
 polkitpolicydir=$(datadir)/polkit-1/actions
 bashcompletiondir=@bashcompletiondir@
@@ -2637,7 +2637,7 @@ udevconfdir = $(sysconfdir)/udev
 dist_udevconf_DATA = \
 	src/udev/udev.conf
 
-sharepkgconfigdir = $(datadir)/pkgconfig
+sharepkgconfigdir = $(libdir)/pkgconfig
 sharepkgconfig_DATA = \
 	src/udev/udev.pc
 
